arch/arm/dts/tegra186.dtsi

```diff
@@ -3,6 +3,8 @@
 #include <dt-bindings/gpio/tegra186-gpio.h>
 #include <dt-bindings/interrupt-controller/arm-gic.h>
 #include <dt-bindings/mailbox/tegra186-hsp.h>
+#include <dt-bindings/interrupt/tegra186-irq.h>
+#include <dt-bindings/clock/tegra186-clock.h>
```

Comment:
This #include is a duplicate of one above. Please keep the includes sorted by filename.